lib/src/controls/yaru_popup_menu_button.dart

```diff
@@ -159,39 +161,44 @@ class _YaruCheckMark extends StatelessWidget {
     final borderColor = Theme.of(context).colorScheme.onSurface.withOpacity(
           Theme.of(context).brightness == Brightness.light ? 0.4 : 1,
         );
-    const size = 18.0;
     const duration = 200;
-    return checked
-        ? AnimatedContainer(
-            height: size,
-            width: size,
-            duration: const Duration(
-              milliseconds: duration,
-            ),
-            decoration: BoxDecoration(
-              color: Theme.of(context).primaryColor,
-              borderRadius: BorderRadius.circular(4),
-            ),
-            child: const Icon(
-              Icons.check,
-              color: Colors.white,
-              size: size,
-            ),
-          )
-        : AnimatedContainer(
-            height: size,
-            width: size,
-            duration: const Duration(
-              milliseconds: duration,
-            ),
-            decoration: BoxDecoration(
-              border: Border.all(
-                color: borderColor,
-                width: 2,
+    return SizedBox(
+      width: _kYaruCheckMarkSize,
+      height: double.infinity,
```

Comment:
had to expand this to full height to be able to center within the expanded height. is there a better way?